src/clients/python/experimental_api_v2/examples/simple_http_v2_string_infer_client.py

```diff
@@ -103,5 +103,23 @@
         if expected_diff[0][i] != r1:
             print("error: incorrect difference")
             sys.exit(1)
+    
+    # Test with an identity model
+    model_name = "savedmodel_zero_1_object"
+    inputs.clear()
+    outputs.clear()
+    inputs.append(httpclient.InferInput('INPUT0', [1, 16], "BYTES"))
+
+    null_chars_array = np.array([u'he\0llo' for i in range(16)], dtype=object)
```

Comment:
Does this need to be \x00 in python 3? https://python-reference.readthedocs.io/en/latest/docs/str/escapes.html

Also should add some tests of just binary arrays being passed as BYTES type for both binary_data true and false. For example just make a byte array of [ 1 ,2, 3, 0, ... ] and put it into a numpy array... did we resolve how byte arrays were best represented in numpy, I recall that there were serveral options.

Reply:
There are several different encoding possible. Similar to API V1 we are exposing the users to specify numpy array of dtype.type=np.bytes_. This will internally traslate to type "|S<byte_size>". 
@CoderHam Is there any other way to simplify the process?